app/src/main/java/org/oppia/android/app/topic/TopicTab.kt

```diff
@@ -9,31 +9,36 @@ enum class TopicTab(
   val positionWithTwoTabs: Int,
   val positionWithFourTabs: Int,
   @StringRes val tabLabelResId: Int,
-  @DrawableRes val tabIconResId: Int
+  @DrawableRes val tabIconResId: Int,
+  @StringRes val contentDescription: Int
```

Comment:
Suggest: ``contentDescriptionResId`` (for consistency with the other vals, and for clarity).

Reply:
done